packages/apps/dev-wallet/src/App/layout-mini.css.ts

```diff
@@ -0,0 +1,32 @@
+import { atoms, tokens, vars } from '@kadena/react-ui/styles';
+import { style } from '@vanilla-extract/css';
+import { globalStyle } from '@vanilla-extract/css';
+
+globalStyle('body', {
```

Comment:
I would move this to a globals overwrite or something

Reply:
sounds better indeed, just moved it